sunpy/io/tests/test_filetools.py

```diff
@@ -44,6 +44,15 @@ def test_read_file_fits(self):
         assert all([isinstance(p[1],
                                sunpy.io.header.FileHeader) for p in pairs])
 
+        # Test file-object
```

Comment:
We should add a test here to check it works as expected with other file objects, something like this:

```python
with open(AIA_171_IMAGE, 'rb') as fd:
    fits_bytes = fd.read()

sunpy.io.read_file(io.BytesIO(fits_bytes))
...
```